README.md

```diff
@@ -17,5 +17,5 @@ https://hub.docker.com/r/hartmutcouk/pg-repack-docker/
     
 ### run (example)
     
-    docker run -e PGPASSWORD=supersecure -it --rm hartmut-co-uk/pg-repack-docker pg_repack -h localhost -U dbroot --dbname=dbname --dry-run --table=table1 --only-indexes --no-superuser-check
+    docker run -it --rm hartmut-co-uk/pg-repack-docker -h localhost -U dbroot --dbname=dbname --dry-run --table=table1 --only-indexes --no-superuser-check
```

Comment:
Why remove the example on how to set the password?
Any particular reason or better alternative?

Reply:
I dropped the example because the change to the entrypoint avoids the containers default postgres startup which fails for me without the password set. This allows pg_repack to prompt for the password. It occurs to me that pg_repack might pick up the password from the environment variable though, so I could go either way.

Reply:
Would you also be able to test by any chance? 
The use case I've had back then for a client was 'scripted' with a cron job - so it was quite essential to not have a prompt.. so I'd like to keep the example for reference. 

What do you think?

Reply:
I went ahead and adjusted the README to include the password-via-environment and also added --network=host on the assumption that one is probably trying to connect to a database on the local development machine when referencing localhost. It does work for me. WDYT?

Reply:
Invariably, my local db has a difference version pf pg_repack and I get this: 

ERROR: pg_repack failed with error: pg_repack 1.4.3 is not installed in the database

In my opinion the primary utility of this container is that pg_repack is very very picky about alignment between postgres and pg_repack versions, and on a normal client machine it can be pretty tricky to get all the right versions worked out to be able to call pg_repack on a server. The docker container locks all that down so it works consistently.